packages/frontend/src/components/ProjectAccess/ProjectAccess.styles.ts

```diff
@@ -26,19 +25,42 @@ export const UserName = styled.b`
     margin-right: 0.625em;
 `;
 
-export const UserEmail = styled(Tag)`
-    width: fit-content;
-    margin-top: 0.3em;
+export const ButtonGroup = styled.div`
+    display: flex;
 `;
 
-export const RoleSelectButton = styled(HTMLSelect)`
-    margin-right: 0.5em;
+export const LoadingArea = styled(Center)`
+    display: flex;
+    flex-direction: column;
 `;
 
-export const AddUserButton = styled(Button)`
-    float: right;
+export const LoadingText = styled(Box)`
+    font-weight: 700;
+    color: #5f6b7c;
+    font-size: 17px;
+    margin-top: 7px;
 `;
 
-export const RelevantOrgRoleIcon = styled(Icon)`
-    margin-right: 10px;
+export const BadgeBox = styled(Box)`
+    min-width: 50px;
+    height: 30px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 5.9px 8.9px;
+    background: #8f99a826;
+    color: black;
+`;
+
+export const EmailBox = styled(Box)`
+    width: fit-content;
+    border-radius: 2px;
+    background: #8f99a826;
+    color: #1c2127;
+    margin-top: 0.3em;
+    min-height: 20px;
+    padding: 2px 6px;
+    font-size: 12px;
+    line-height: 16px;
+    box-sizing: border-box;
```

Comment:
Hi @Ankush263, thank you for your contribution! As we're transitioning from Blueprint to Mantine, our aim is to eliminate the use of styled component files. I noticed that this file introduces a significant amount of custom CSS, including hardcoded colors and values. While your efforts are appreciated, we're looking to avoid such customizations moving forward. Would you be able to revise this?

Reply:
I can fix these but some of the colour components and styles are going to change as Mantile components have different styles than Blueprintjs. So, some of the components look different from the previous. 
Also, if we want to eliminate the style component files, we have to use `sx` property in Mantine as we might need to change some default styling in Mantine in some cases.
If you are okay with that, I can refactor this branch😀

Reply:
Got it! I will work on restructuring this PR.🚀